pallets/confidential/Cargo.toml

```diff
@@ -30,30 +30,20 @@ sp-runtime-interface = { git = "https://github.com/paritytech/substrate", defaul
 # Crypto
 rand_core = { version = "0.5", default-features = false }
 rand = { version = "0.7.3", default-features = false, optional = true }
-cryptography = { git = "https://github.com/PolymathNetwork/cryptography.git", branch = "develop", default-features = false }
-curve25519-dalek = { git = "https://github.com/PolymathNetwork/curve25519-dalek.git", branch = "v2-packed-simd", default-features = false, features = ["nightly"] }
-bulletproofs = { git = "https://github.com/PolymathNetwork/bulletproofs.git", branch = "v2-packed-simd", default-features = false, features = ["zeroize"] }
+cryptography_core = { git = "https://github.com/PolymathNetwork/cryptography.git", branch = "CRYP-187/cleanup-dir-structure_miguel", default-features = false }
```

Comment:
I can't wait for this `confidential` pallet to be removed. It's essentially accessing the bulletproofs library via the cryptography library... And it's dead code...

Reply:
Actually @ParnianAlimi you could delete it. This pallet was created as a E2E test for the integration with crypto-library.
In order to delete it, pls be aware that you should migrate its benchmarks too.

Reply:
Yeah, I brought this up a while ago. and Adam said it's for the future.